docs/kafka/metrics-monitoring-kafka/README.adoc

```diff
@@ -201,9 +201,11 @@ As an alternative to viewing metrics for a Kafka instance in the {product-long-k
 This procedure follows the https://prometheus.io/docs/prometheus/latest/configuration/configuration/#configuration-file[Configuration File^] method defined by Prometheus for integrating third-party metrics. If you use the Prometheus Operator in your monitoring environment, you can also follow the https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/additional-scrape-config.md#additional-scrape-configuration[Additional Scrape Configuration^] method.
 
 .Prerequisites
-* You have access to a running Kafka instance that contains topics in {product-kafka}. For more information about access management in {product-kafka}, see {base-url}{access-mgmt-url-kafka}[Managing account access in {product-long-kafka}^].
-* You have the ID and the SASL/OAUTHBEARER token endpoint for the Kafka instance. To relocate the Kafka instance ID and the token endpoint, select your Kafka instance in the {product-kafka} web console, select the options menu (three vertical dots), and click *Connection*.
-* You have the generated credentials for your service account that has access to the Kafka instance. To reset the credentials, use the {service-accounts-url}[Service Accounts^] page in the *Application Services* section of the Red Hat Hybrid Cloud Console.
+* You've access to a running Kafka instance that contains topics in {product-kafka}. For more information about access management in {product-kafka}, see {base-url}{access-mgmt-url-kafka}[Managing account access in {product-long-kafka}^].
+* You've the ID and the SASL/OAUTHBEARER token endpoint for the Kafka instance. To relocate the Kafka instance ID and the token endpoint, select your Kafka instance in the {product-kafka} web console, select the options menu (three vertical dots) and click: 
+** *Details* to locate the ID which is the ID of the Kafka instance.
+** *Connection* tab to locate the SASL/OAUTHBEARER Token endpoint URL.
```

Comment:
```suggestion
** *Connection* tab to locate the SASL/OAUTHBEARER token endpoint URL.
```

Reply:
@AedinC I used upper case "T" for token because that is the way it appears in the UI